OpenSourcePolitics/decidim-app
# Add force SSL env var

#### :tophat: Description
Add a new ENV variable to enable / disable the force SSL configuration.  
This is needed for reverse proxy setup where Decidim is only launched on HTTP port (80) and the HTTPS access is done elsewhere (load balancer, external proxy ...)

#### Testing

Force SSL enable (Standard behavior) : 
* **Do not** set the environment variable
* Launch Decidim in production mode
* `curl` or access your decidim in a browser with `http://`
* You should be redirected to `https://`

Force SSL disable
* Set the environment variable FORCE_SSL to 0
* Launch Decidim in production mode
* `curl` or access your decidim in a browser with `http://`
* You should **not** be redirected to `https://` and acces directly your decidim through HTTP
